sa/storage-authority.go

```diff
@@ -869,3 +881,32 @@ func (ssa *SQLStorageAuthority) AddSCTReceipt(sct core.SignedCertificateTimestam
 	}
 	return err
 }
+
+func hashNames(names []string) []byte {
+	names = core.UniqueLowerNames(names)
+	hash := sha256.Sum256([]byte(strings.Join(names, ",")))
+	return hash[:]
+}
+
+func addFQDNSet(tx *gorp.Transaction, names []string, serial string, issued time.Time, expires time.Time) error {
+	return tx.Insert(&core.FQDNSet{
+		SetHash: hashNames(names),
+		Serial:  serial,
+		Issued:  issued,
+		Expires: expires,
+	})
+}
+
+// CountValidFQDNSets reutrns the number of currently valid sets with hash |setHash|
```

Comment:
Update the method name and comment to reflect that this is no longer just "valid" (i.e. unexpired).